docs/FilteringTutorial.md

```diff
@@ -60,13 +60,13 @@ React-admin offers 4 different ways to filter the list. Depending on the type of
 </video>
 
 
-The default appearance for filters is an inline form displayed on top of the list. Users also see a dropdown button allowing to add more inputs to that form. This functionality relies on the `<List filters>` prop: 
+The default appearance for filters is an inline form displayed on top of the list. Users also see a dropdown button allowing to add more inputs to that form. This functionality relies on the `<List filters>` prop:
 
 ```jsx
-import { TextInput } from 'react-admin';
+import { TextInput, SearchInput } from 'react-admin';
 
 const postFilters = [
-    <TextInput label="Search" source="q" alwaysOn />,
+    <SearchInput source="q" alwaysOn />,
```

Comment:
I'd keep the `TextInput` here since we introduce the `SearchInput` just a few lines below.